README.md

```diff
@@ -142,7 +144,7 @@ waterwheel.api.user.get(1)
   });
 ```
 `.get()` accepts two arguments
-  - `token`: The token of the entity you are requesting, `nid`.
+  - `identifier`: The identifier of the entity you are requesting, `nid`.
```

Comment:
Should you add, "ie. " to be consistent with other docs and let users know the internal identifier on Drupal side will change?